xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -498,9 +498,10 @@ int CPVRChannelGroup::GetMembers(CFileItemList &results, bool bGroupMembers /* =
 
   for (PVR_CHANNEL_GROUP_SORTED_MEMBERS::const_iterator it = channels->m_sortedMembers.begin(); it != channels->m_sortedMembers.end(); ++it)
   {
-    if (bGroupMembers || !IsGroupMember((*it).channel))
+    const CPVRChannelPtr channel = (*it).channel;
```

Comment:
replace the for loop with a range based for loop

Reply:
not in scope of this PR

Reply:
This is already a non functional change, then why not do it correct?